Stack/Stack/Stack.c

```diff
@@ -0,0 +1,38 @@
+#include "Stack.h"
+#include <stdio.h>
+#include <malloc.h>
+
+bool isEmpty(Stack* head)
+{
+    return head == NULL;
+}
+
+void push(Stack** head, int element)
+{
+    Stack* newStack = (Stack*)calloc(1, sizeof(Stack));
+    if (newStack == NULL)
+    {
+        return;
+    }
+    newStack -> value = element; 
```

Comment:
```suggestion
    newStack->value = element; 
```
Обращение к полю никогда пробелами не выделяется, хотя, казалось бы, это бинарный оператор, так что должно бы